package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/ script/",
     "foreman-js:link": "./script/npm_link_foreman_js.sh",
     "postlint": "./script/npm_lint_plugins.js",
-    "test": "node --max_old_space_size=8192 node_modules/.bin/jest --no-cache --config config/jest.config.js",
+    "test": "jest",
```

Comment:
Can't we specify `jest --config config/jest.config.js` to keep our root directory a bit cleaner?